src/content/plugins/module-concatenation-plugin.md

```diff
@@ -16,3 +16,78 @@ new webpack.optimize.ModuleConcatenationPlugin()
 ```
 
 > Scope Hoisting is specifically a feature made possible by ECMAScript Module syntax. Because of this webpack may fallback to normal bundling based on what kind of modules you are using, and [other conditions](https://medium.com/webpack/webpack-freelancing-log-book-week-5-7-4764be3266f5).
+
+# Optimization Bailouts
+
+As the article explains, there Webpack attempts to achieve partial scope hoisting. Webpack attempts to merge modules into a single scope but cannot do so in every case. If webpack cannot merge a module, the two alternatives are Prevent and Root. Prevent means the module must be in its own scope. Root means the module begins a new module group. The following table explains the conditions:
```

Comment:
As per the [branding guidelines](https://github.com/webpack/media), "Webpack" should be always be written in lowercase.

Reply:
Thanks!